devfile.yaml

```diff
@@ -11,30 +11,31 @@ schemaVersion: 2.1.0
 metadata:
   name: che-code
 components:
+
   - name: dev
     container:
-      image: quay.io/che-incubator/che-code-dev:insiders
+      image: quay.io/che-incubator-pull-requests/che-code-dev:pr-246-dev-amd64
       memoryLimit: 12Gi
+      memoryRequest: 512Mi
+      cpuRequest: 500m
       cpuLimit: 3500m
-      endpoints:
-        - exposure: public
-          name: dev
-          secure: true
-          protocol: http
-          targetPort: 8000
+      mountSources: true
+
   - name: projects
     volume:
-      size: 3Gi
+      size: 7Gi
+
 commands:
-  - id: prepare
```

Comment:
If the purpose of adding `yarn-` prefix is for UI only, you can add [`exec.label` property](https://github.com/eclipse/che/issues/21625) with the descriptive command name.

Reply:
Done with that